BuildToolsVersion.txt

```diff
@@ -1 +1 @@
-1.0.25-prerelease-00182
+1.0.25-prerelease-00182
```

Comment:
If I recall, any extra whitespace in this file can cause build issues.  Whichever one we pick, we want an exact match with CoreFx.

---
In reply to: [54840341](https://github.com/dotnet/wcf/pull/872#discussion_r54840341) [](ancestors = 54840341)